src/platform-service/templates/deployment.yaml

```diff
@@ -73,7 +73,7 @@ spec:
         - configMapRef:
             name: platform-service
         - secretRef:
-            name: smtpSecret
+            name: smtp-secret
```

Comment:
smtpSecret to smtp-secret is just cosmetic right ?
is this name referenced anywhere ?